### PR TITLE
Settings Rework

### PR DESCRIPTION
**Pull request by jannuary**
# Description
Rearranges and unifies several bottle settings.

Fixes https://github.com/bottlesdevs/Bottles/issues/2246.

image

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

### CODE REVIEW

**Comment by jannuary** [comment]
likely left something out, but here goes

File: bottles/frontend/ui/details-preferences.blp [hunk 1]
```diff
@@ -68,38 +106,17 @@ template DetailsPreferences : .AdwPreferencesPage {
       model:
       StringList {
         strings [
+          _("Disabled"),
           _("Ultra Quality"),
           _("Quality"),
           _("Balanced"),
           _("Performance"),
         ]
```
Comment (TheEvilSkeleton) [inline bottles/frontend/ui/details-preferences.blp:114]: There are [5 levels](https://support.system76.com/articles/use-amd-fsr/#resolution-and-quality) for FSR, but there are only 3 here. I added "Ultra Performance", but we'll need another one to increment. What do you suggest?

```suggestion
        strings [
          _("Disabled"),
          _("Ultra Quality"),
          _("Quality"),
          _("Balanced"),
          _("Performance"),
          _("Ultra Performance"),
        ]
```

**Review by TheEvilSkeleton — CHANGES_REQUESTED** [review]

File: bottles/frontend/ui/details-preferences.blp (hunk 1 above)
Comment (jannuary) [reply bottles/frontend/ui/details-preferences.blp:114]: Out of scope for this specifically :)

File: bottles/frontend/ui/details-preferences.blp [hunk 2]
```diff
@@ -335,66 +301,15 @@ template DetailsPreferences : .AdwPreferencesPage {
...
-
-      Switch switch_vmtouch {
-        valign: center;
-      }
-    }
-
     .AdwActionRow row_cwd {
       title: _("Working Directory");
-
+      activatable-widget: btn_cwd;
```
Comment (TheEvilSkeleton) [inline bottles/frontend/ui/details-preferences.blp:312]: Adwaita Demo and Bottles don't do that. It's best to remove it.

```suggestion
```

**Review by TheEvilSkeleton — CHANGES_REQUESTED** [review]

File: bottles/frontend/ui/details-preferences.blp [hunk 3]
```diff
@@ -2,48 +2,86 @@ using Gtk 4.0;
...
+
+        model:
+          StringList str_list_runner {};
       }
 
-      Switch switch_dxvk {
-        valign: center;
+      .AdwComboRow combo_dxvk {
+        title: _("DXVK");
+        subtitle: _("Increase Direct3D compatibility by translating it to Vulkan.");
```
[the remark this answers is not in the document]
Comment (TheEvilSkeleton) [reply bottles/frontend/ui/details-preferences.blp:28]: @jannuary you missed this ^

File: bottles/frontend/ui/details-preferences.blp (hunk 2 above)
Comment (jannuary) [reply bottles/frontend/ui/details-preferences.blp:312]: No, let's not - whichever other place should have an activatable-widget too. Besides, I'm not too sure how much the demo should be used for *design* reference.

Comment (TheEvilSkeleton) [reply bottles/frontend/ui/details-preferences.blp:312]: Sure. There's only one thing to discuss about.

File: bottles/frontend/ui/details-preferences.blp (hunk 3 above)
[the remark this answers is not in the document]
Comment (jannuary) [reply bottles/frontend/ui/details-preferences.blp:28]: I am unsure of the relevancy of the versions in the first place, and it seems clear that the exception to the generic DXVK option doesn't in fact apply to the same thing as the generic DXVK option.

[the remark this answers is not in the document]
Comment (TheEvilSkeleton) [reply bottles/frontend/ui/details-preferences.blp:28]: It's not really clear to me honestly. If I didn't know then I'd assume it's for Direct3D 8 and below.

**Pull request merged**